ee/clickhouse/materialized_columns/columns.py

```diff
@@ -40,15 +38,36 @@
 }
 
 
-class MaterializedColumn(NamedTuple):
+@dataclass
+class MaterializedColumn:
     name: ColumnName
     details: MaterializedColumnDetails
```

Comment:
I'll probably consolidate these two classes at some point (there's not much benefit to them being separate at this point) but not in a big hurry to do that.